src/desktop/components/search_bar/view.coffee

```diff
@@ -118,13 +118,30 @@ module.exports = class SearchBarView extends Backbone.View
 
   # Display spotlight result if:
   # - more than 4 characters have been typed AND
-  # - query is an exact case-insensitive leading match of the search result AND
-  # - query is more than 30% of the length of the search result
+  # - query is more than 30% of the length of the search result AND
+  # - query is a case-insensitive leading match of the search result or any token
   shouldShowSpotlightSearch: (searchResult) ->
+    @isSubsequentMatch = false
     currentQuery = @$input.val()
     currentQuery.length > 4 &&
-      searchResult.toLowerCase().indexOf(currentQuery.toLowerCase()) is 0 &&
-      (currentQuery.length / searchResult.length) >= 0.3
+      (currentQuery.length / searchResult.length) >= 0.3 &&
+      @isLeadingMatchOfAnyToken(currentQuery, searchResult)
+
+  # Return true if:
+  #  - result starts with word OR
+  #  - any token in result starts with word (sets isSubsequentMatch to true)
+  isLeadingMatchOfAnyToken: (word, result) ->
+    return true if result.toLowerCase().indexOf(word.toLowerCase()) is 0
+
+    if word.indexOf(' ') > -1
+      regex = new RegExp(word, 'i')
+      return @isSubsequentMatch = true if result.match(regex)
```

Comment:
one thing we drop with this is exact matches on spelling errors... but i think that's totally fine as we don't always promote correctly to the top spot in the auto-suggest till the very end of the query in those cases (eg try `damian hirst` or `jef koons`)

Reply:
oo good point about spelling errors!